static/app/components/workflowEngine/gridCell/actionCell.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from '@emotion/styled';
+
+import {IconCircledNumber} from 'sentry/components/iconCircledNumber';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconMail} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import PluginIcon from 'sentry/plugins/components/pluginIcon';
+import {space} from 'sentry/styles/space';
+
+const ActionMetadata = {
+  slack: {name: t('Slack'), icon: <PluginIcon pluginId="slack" size={20} />},
+  discord: {name: t('Discord'), icon: <PluginIcon pluginId={'discord'} size={20} />},
+  email: {name: t('Email'), icon: <IconMail size="md" />},
+};
+
+export type Action = keyof typeof ActionMetadata;
+
+type ActionCellProps = {
+  actions: Action[];
+};
+
+export function ActionCell({actions}: ActionCellProps) {
+  if (!actions || actions.length === 0) {
+    return <div>{t('No actions')}</div>;
+  }
+  if (actions.length === 1 && actions[0]) {
+    return (
+      <Inline>
+        <IconContainer>{ActionMetadata[actions[0]].icon}</IconContainer>
+        {ActionMetadata[actions[0]].name}
+      </Inline>
+    );
+  }
+  const actionsList = actions
+    .map(action => ActionMetadata[action].name)
+    .reduce((acc, action) => `${acc}, ${action}`);
+  return (
+    <Inline>
+      <IconContainer>
+        <IconCircledNumber number={actions.length} />
+      </IconContainer>
+      <Tooltip title={actionsList}>
```

Comment:
I'm happy to implement this as a follow-up, but just a heads up that we'll probably use a `ResizeObserver` to track if the actions list is truncated and only display the tooltip if it is. Pretty sure we use that pattern elsewhere. Just makes the UX a little more thoughtful.

Reply:
I'm thinking we can add this once we put the cell components into a table? Right now the `GridEditable` component I'm using for the stories wraps the text instead of truncating, which makes it hard to test using a `ResizeObserver`

Reply:
Definitely fine to skip for now